source/posix/socket.c

```diff
@@ -975,7 +975,7 @@ int aws_socket_shutdown_dir(struct aws_socket *socket, enum aws_channel_directio
  * 1st scenario, someone called write and we want to at least try and make a write call if we can or return an
  * error if something bad has happened to the socket.
  * 2nd scenario, the event loop notified us that the socket went writable. */
-static int s_process_write_requests(struct aws_socket *socket, bool spawned_from_event) {
+static int s_process_write_requests(struct aws_socket *socket, struct write_request *spawned_from_request) {
```

Comment:
Any reason not to use parent_request nomenclature instead? Makes some of the logic easier to follow, at least for me.

Reply:
sure, I was using an arg name similar to the one it was replacing, but I admit it was awkward